src/test/java/api/requests/RequestsAPICreationTests.java

```diff
@@ -1625,6 +1627,104 @@ private void recallItem(ItemResource item, List<UserResource> users) {
       .forEach(index -> requestsFixture.recallItem(item, users.get(index)));
   }
 
+  @Test
```

Comment:
Can we just add an item in status `Available` to the next test, and remove this one?

Reply:
Tests are refactored as per agreement, thanks.